lib/takeaway.rb

```diff
@@ -0,0 +1,73 @@
+require 'twilio-ruby'
+
+class Order
+attr_reader :menu, :order, :total, :total_items, :meal_choice, :meal_count, :total_cost
+
+def initialize
+  @menu = {Pizza: 10, Pasta: 7.50, Cheesesteak: 8.50, Fries: 3.50, Donut: 2.50}
+  @order = []
+  @total_cost = 0
+  @total_items = 0
+end
+
+def take_order
```

Comment:
(from rubric) > You may find it difficult to remember the indentation rules, but one helpful rule of thumb for indentation is to ensure that you are two space indented inside any do ... end, class ... end or def ... end block.